layouts/_default/baseof.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html lang="{{ .Site.LanguageCode }}">
+<html lang="{{ .Site.LanguageCode }}" class="direction-{{ .Language.LanguageDirection | default "ltr" }}"> 
+<!-- https://github.com/adityatelange/hugo-PaperMod/pull/47 -->
```

Comment:
Don't include a link to another GitHub repo in the header. If that was your inspiration for this part of the code, include it in PR's description.

Reply:
Done